src/main/scala/org/constellation/primitives/RandomTransactionManager.scala

```diff
@@ -211,4 +211,146 @@ class RandomTransactionManager[T](nodeActor: ActorRef, periodSeconds: Int = 10)(
       "randomTransactionLoop"
     )
   }
+
+  def generate(): Unit =
```

Comment:
I think the type is invalid because you're returning the for..comp which has type `F[Unit]` or `EitherT[F, ?, Unit]`, I'm not sure which one